src/components/Form/FormItem.tsx

```diff
@@ -0,0 +1,115 @@
+import { Form as AntForm, FormItemProps as AntFormItemProps, FormInstance } from 'antd'
+import { CSSProperties, ReactElement, ReactNode, isValidElement, useMemo } from 'react'
+
+import { Checkbox } from '../Checkbox'
+import { Input } from '../Input'
+import { RadioGroup } from '../RadioGroup'
+import { Switch } from '../Switch'
+
+type RenderProps = {
+  form: FormInstance,
+  value: unknown;
+  onChange: (value: unknown) => void
+}
+export type FormItemProps = {
+  name?: string
+  label?: string
+  style?: CSSProperties
+  span?: number
+  isFullWidth?: boolean,
+  rules?: AntFormItemProps['rules']
+  valuePropName?: AntFormItemProps['valuePropName']
+  getValueFromEvent?: AntFormItemProps['getValueFromEvent']
+  children: ReactElement | ((props: RenderProps) => ReactNode)
+}
```

Comment:
props are not documented. Could we create a dedicated types file with JSDocs?

Reply:
yes, we will document the props in the next implementation step

Reply:
props documented here https://github.com/mia-platform/design-system/blob/IDP-94/form-analysis/src/components/Form/Validators/index.ts